src/wire-protocol.ts

```diff
@@ -1,11 +1,60 @@
+import { fql } from "./query-builder";
+import { TaggedTypeEncoded } from "./tagged-type";
+
+/**
+ * A piece of an interpolated query that represents an actual value. Arguments
+ * are passed to fauna using ValuFragments so that query string injection is not
+ * possible.
+ * @remarks A ValueFragment is created by this driver when a literal value or
+ * object is provided as an argument to the {@link fql} tagged template
+ * function.
+ *
+ * ValueFragments must always be encoded with tags, regardless of the "x-format"
+ * request header sent.
+ * @example
+ * ```typescript
+ *  const num = 17;
+ *  const query = fql`${num} + 3)`;
+ *  // produces
+ *  { fql: [{ value: { "@int": "17" } }, " + 3"] }
+ * ```
+ */
+export type ValueFragment = { value: TaggedTypeEncoded };
+
+/**
+ * A piece of an interpolated query. Queries can be safely composed together
```

Comment:
```suggestion
 * A piece of an interpolated query. Interpolated queries can be safely composed together
```

Reply:
I like it 👍🏻

Reply:
we'll have to regen lib once suggestions are merged 😵‍💫

Reply:
done (and new build too 😆)